grafana-plugin/e2e-tests/outgoingWebhooks/createAdvancedWebhook.test.ts

```diff
@@ -0,0 +1,46 @@
+import { test } from '../fixtures';
+import { clickButton, generateRandomValue } from '../utils/forms';
+import { createIntegration } from '../utils/integrations';
+import { goToOnCallPage } from '../utils/navigation';
+import { checkWebhookPresenceInTable } from '../utils/outgoingWebhooks';
+
+test('create advanced webhook and check it is displayed on the list correctly', async ({ adminRolePage: { page } }) => {
+  const WEBHOOK_NAME = generateRandomValue();
+  const WEBHOOK_INTEGRATION_NAME = generateRandomValue();
+  const WEBHOOK_URL = 'https://example.com';
+
+  await createIntegration({ page, integrationSearchText: 'Webhook', integrationName: WEBHOOK_INTEGRATION_NAME });
+
+  await goToOnCallPage(page, 'outgoing_webhooks');
+
+  await clickButton({ page, buttonText: 'New Outgoing Webhook' });
+  await page.getByText('Advanced').first().click();
+  await page.waitForTimeout(2000);
+
+  const webhooksForm = page.locator('#OutgoingWebhook div');
```

Comment:
is this a div generated by some 3rd party like grafana-ui or can we append a testid to it? Seems more test-friendly to use a test id instead

Reply:
This is actually to select divs within webhook form which are later filtered by the text they include. I renamed the const to make it more clear